pytest/tests/mocknet/helpers/load_test_spoon_helper.py

```diff
@@ -4,267 +4,118 @@
-            lambda: account.send_call_contract_raw_tx(
-                next_account_id, 'delete_state', s.encode('utf-8'), 0))
-        logger.info(
-            f'{account.key.account_id} delete_state on {next_account_id} {tx_res}'
-        )
-        if item in function_call_state[i]:
-            function_call_state[i].remove(item)
-            logger.info(
-                f'Successfully removed {item} from function_call_state. New #items: {len(function_call_state[i])}'
-            )
-        else:
-            logger.info(
-                f'{item} is not in function_call_state even though this is impossible. #Items: {len(function_call_state[i])}'
-            )
-
-
-def function_call_ft_transfer_call(account, i, node_account):
-    next_id = random.randint(0, mocknet.NUM_ACCOUNTS - 1)
-    dest_account_id = mocknet.load_testing_account_id(
-        node_account.key.account_id, next_id)
-
-    s = f'{{"receiver_id": "{dest_account_id}", "amount": "3", "msg": "\\"hi\\""}}'
-    logger.info(
-        f'Calling function "ft_transfer_call" with arguments {s} on account {account.key.account_id} contract {dest_account_id}'
-    )
-    tx_res = mocknet_helpers.retry_and_ignore_errors(
-        lambda: account.send_call_contract_raw_tx(
-            dest_account_id, 'ft_transfer_call', s.encode('utf-8'), 1))
-    logger.info(
-        f'{account.key.account_id} ft_transfer to {dest_account_id} {tx_res}')
-
-
-QUERIES_PER_TX = 5
-
-
-def random_transaction(account, i, node_account, max_tps_per_node):
-    time.sleep(random.random() * mocknet.NUM_ACCOUNTS / max_tps_per_node / 3)
-    choice = random.randint(0, 2)
-    if choice == 0:
-        logger.info(f'Account {i} transfers')
-        send_transfer(account, node_account)
-    elif choice == 1:
-        function_call_set_delete_state(account, i, node_account)
-    elif choice == 2:
-        function_call_ft_transfer_call(account, i, node_account)
-
-
-# rpc_infos is not needed in this file, but it's present here for compatibility with load_test_betanet.py.
-def send_random_transactions(node_account,
-                             test_accounts,
-                             max_tps_per_node,
-                             rpc_infos=None):
-    pmap(
-        lambda index_and_account: random_transaction(index_and_account[
-            1], index_and_account[0], node_account, max_tps_per_node),
-        enumerate(test_accounts))
-
-
-def write_tx_events(accounts_and_indices, filename):
-    # record events for accurate input tps measurements
-    all_tx_events = []
-    for (account, _) in accounts_and_indices:
-        all_tx_events += account.tx_timestamps
-    all_tx_events.sort()
-    with open(filename, 'w') as output:
-        for t in all_tx_events:
-            output.write(f'{t}\n')
-
-
 def get_test_accounts_from_args(argv):
     node_account_id = argv[1]
-    rpc_nodes = argv[2].split(',')
-    num_nodes = int(argv[3])
-    max_tps = float(argv[4])
+    pk = argv[2]
```

Comment:
should revert this part here to what was added in https://github.com/near/nearcore/pull/8659, otherwise the args that `mocknet.py` will set for this helper script will be wrong

Reply:
Done